homeassistant/components/iotawatt/config_flow.py

```diff
@@ -98,6 +109,36 @@ async def async_step_auth(self, user_input=None):
 
         return self.async_create_entry(title=data[CONF_HOST], data=data)
 
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Get the options flow for this handler."""
+        return OptionsFlow(config_entry)
+
+
+class OptionsFlow(config_entries.OptionsFlow):
+    """Handle a config flow options for iotawatt."""
+
+    def __init__(self, config_entry):
+        """Initialize."""
+        self._data = dict(config_entry.data)
+
+    async def async_step_init(self, user_input: dict[str, Any] = None):
+        """Handle the initial step."""
+        if user_input is not None:
+            self._data.update(user_input)
+            return self.async_create_entry(title="", data=self._data)
+
+        data_schema = vol.Schema(
+            {
+                vol.Optional(
+                    CONF_SCAN_INTERVAL,
+                    default=self._data.get(CONF_SCAN_INTERVAL, 30),
+                ): vol.All(int, vol.Range(min=5)),
+            }
+        )
```

Comment:
We no longer allow to provide a configuration option for this. Instead, one can disable the polling updates and use the service  `homeassistant.entity_update` using automations instead.

Reply:
Will do, to clarify it is `homeassistant.update_entity`